Fixes the following configuration:
$ ./setup configure --ghc --prefix=/usr --with-compiler=/usr/bin/ghc --with-hc-pkg=/usr/bin/ghc-pkg --prefix=/usr --libdir=/usr/lib64 --libsubdir=git-annex-5.20151102.1/ghc-7.8.4 --datadir=/usr/share/ --datasubdir=git-annex-5.20151102.1/ghc-7.8.4 --ghc-option=-j4 --ghc-option=-optc-march=native --ghc-option=-optc-O2 --ghc-option=-optc-pipe --ghc-option=-optl-Wl,-O1 --ghc-option=-optl-Wl,--as-needed --disable-executable-stripping --docdir=/usr/share/doc/git-annex-5.20151102.1 --verbose --sysconfdir=/etc --disable-library-stripping --flag=-android --flag=-androidsplice --flags=asciiprogress --flags=-assistant --flags=database --flags=-Cryptonite --flags=dbus --flags=desktopnotify --flags=dns --flags=-ekg --flags=feed --flags=inotify --flags=network-uri --flags=pairing --flag=-production --flags=-quvi --flags=-s3 --flags=-tahoe --flags=tdfa --flag=-testsuite --flags=-torrentparser --flags=-webapp --flags=-webapp-secure --flags=-webdav --flags=-xmpp

Utility/Url.hs:45:0:
     error: missing binary operator before token "("
     #if ! MIN_VERSION_http_client(0,4,18)
     ^
diff --git a/git-annex.cabal b/git-annex.cabal
index 2a71b2c..ba1af9e 100644
--- a/git-annex.cabal
+++ b/git-annex.cabal
@@ -131,3 +131,3 @@ Executable git-annex
    resourcet, http-conduit, http-types,
-   time, old-locale
+   time, old-locale, http-client
   CC-Options: -Wall
